docs/README.md

```diff
@@ -33,42 +37,736 @@ All output will adhere to the same general JSON structure:
+
+
+```http
+GET /auth/reset/[reset-token]
+```
+
+#### Body
+A valid token
+```json
+{
+  "email": "rijk@example.com",
+  "instance": "https://example.com/admin/"
+}
+```
+
+#### Possible Results
+| Code             | Description                                        |
+|------------------|----------------------------------------------------|
+| 200 OK           | Always returns success to avoid malicious checks for valid email addresses     |
+
+
 
-### Users
 
-### Server-use Tokens
 
 
 ## Items
 
-### Get Items
 
-Gets the items from a given collection.
+
+
+### Create Item
+Creates one or more items within a given collection
 
 ```http
-/items/<collection>
+POST /items/[collection-name]/
 ```
 
-#### Supported parameters
+#### Body
+A single item or an array of multiple items to be created. Field keys must match the collection's column names.
+
+#### Possible Results
+| Code                     | Description                                        |
+|--------------------------|----------------------------------------------------|
+| 201 Created              | `data`: The created item(s), including default fields added by Directus  |
+| 400 Bad Request          | `message`: Syntax error in provided JSON    |
+| 404 Not Found            | `message`: Collection doesn’t exist    |
+| 422 Unprocessable Entity | `message`: Column doesn’t exist in collection    |
 
-| param | description                      | default value |
-|-------|----------------------------------|---------------|
-| limit | The number of records to request | 20            |
-| sort  | What field to sort by            | id            |
-| q     | String to search by              | --            |
+
+::: tip
+The API may not return any data for successful requests if the user doesn't have adequate read permission
+:::
+
+
+
+
+### Get Item
+Gets a single item from within a given collection
+
+```http
+GET /items/[collection-name]/[primary-key-value]
+```
+
+#### Query Parameters
+| Name          | Default   | Description                                      |
+|---------------|-----------|--------------------------------------------------|
+| fields        | *         | CSV of fields to include in response [Learn More](#fields)  |
+| meta          |           | CSV of metadata fields to include [Learn More](#metadata)  |
+| status        | Published | CSV of statuses [Learn More](#status)  |
+| lang          | *         | Include translation(s) [Learn More](#language)  |
+
+#### Possible Results
+| Code             | Description                                        |
+|------------------|----------------------------------------------------|
+| 200 OK           | `data`: Retrieved item<br>`meta`: Depends on requested metadata  |
+| 404 Not Found    | `message`: Collection doesn’t exist, or item doesn't exist within collection    |
+
+#### Examples
+
+* Return the project item with an ID of `1`
+  ```bash
+  curl -g https://api.directus.io/_/items/projects/1
```

Comment:
The `-g` flag in `curl` is only needed when you want to use brackets in query parameters like filter **[** name **]** **[** eq **]** =rijk